app/containers/markdown/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+export interface UserMention {
+	_id: string;
+	username: string;
+	name?: string;
+}
+
+export interface UserChannel {
+	[index: number]: string | number;
+	name: string;
+	_id: string;
+}
+
+export type TOnLinkPress = (link: string) => void;
```

Comment:
Usually a file that concentrates the types we call "types" (including vscode understands this alias) and not interfaces.
Why not call this file types.ts?

Reply:
We started to do the files that handle interfaces or types like this. But, WDYT?

Reply:
I think it's better to use the pattern that the community uses hahah.
But in this case, leave it that way, in the next few I think it's cool to use this pattern :)